AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -74,46 +77,15 @@ private extension SessionHeaderView {
         .foregroundColor(.darkBlue)
     }
     
-    var measurementsTitle: some View {
-        Text("Most recent measurement:")
-    }
-    
-    var measurementsAB: some View {
-        Group {
-            if let measurements = extractLatestMeasurements() {
-                VStack(alignment: .leading, spacing: 5) {
-                    measurementsTitle
-                    HStack {
-                        Group {
-                            singleMeasurement(streamName: "PM1", value: measurements.pm1)
-                            singleMeasurement(streamName: "PM2", value: measurements.pm25)
-                            singleMeasurement(streamName: "PM10", value: measurements.pm10)
-                            singleMeasurement(streamName: "F", value: measurements.f)
-                            singleMeasurement(streamName: "RH", value: measurements.h)
-                        }
-                        .frame(maxWidth: .infinity)
-                    }
-                }
-            } else {
-                VStack(alignment: .leading, spacing: 3) {
-                    Text("Your AirBeam is gathering data.")
-                        .font(Font.moderate(size: 14))
-                    Text("Measurements will appear in 3 minutes.")
-                        .font(Font.moderate(size: 12))
-                }
-                .foregroundColor(.darkBlue)
-            }
-        }
-    }
-    
-    func thresholdFor(name: String) -> SensorThreshold? {
-        thresholds.first { $0.sensorName == name }
-    }
-    
     var measurementsMic: some View {
         VStack(alignment: .leading, spacing: 5) {
-            measurementsTitle
-            singleMeasurement(streamName: "db", value: lastMicMeasurement())
+            Text("Most recent measurement:")
+            if let dbStream = session.dbStream {
```

Comment:
I'm no expert in SwiftUI, but will this work `guard let dbStream = session.dbStream else { return }`? If yes we can get rid of one level of indentation here.

Reply:
Unfortunately, it won't work

Reply:
Unfortunately, no. Guard cannot be used inside SwiftUI view builder.

Reply:
Well, my first steps in swiftui will be painful for sure 😬